README.md

```diff
@@ -60,4 +57,4 @@ copier copy https://github.com/Ingenjorsarbete-For-Klimatet/ifk-python-template.
 ```
 
 Copier supports updating projects based on templates when the template updates.
-See <a href="https://copier.readthedocs.io/en/stable/" taget="_blank">copier docs</a>.
\ No newline at end of file
+See <a href="https://copier.readthedocs.io/en/stable/" taget="_blank">copier docs</a>.
```

Comment:
I must be blind, but what is changed here? Is it a new line at the end of the file?

Reply:
Beats me. Haven't made any conscious changes to the line in any case.